examples/magento-graphcms/pages/[...url].tsx

```diff
@@ -77,6 +77,7 @@ function CategoryPage(props: Props) {
           title={page?.metaTitle ?? ''}
           metaDescription={page?.metaDescription ?? ''}
           metaRobots={page?.metaRobots.toLowerCase().split('_').flat(1) as MetaRobots[]}
+          canonical={`/${page?.url}`}
```

Comment:
A page doesn't always have a URL, because it can be that there is no page defined in GraphCMS, so we need to check for it.

```suggestion
          canonical={page?.url ? `/${page?.url}` : undefined}
```

You might want to use category.url_path instead since that one will always exist